### PR TITLE
Add comm article with an image. Replaces #3104, which does not have t…

### PR DESCRIPTION
**Pull request by RackerKate**
…he image: INFODEV-3537

### CODE REVIEW

**Comment by RackerKate** [comment]
Still waiting on SME clarification.

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-1c5198644e/how-to/). :bow:

**Comment by rackernexus** [comment]
Build 85 was successful. Staging content [Preview!](https://developer-revs.staging.deconst.org/build-589f40de6/docs/). valid for 14 days

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-6d6c726a30/how-to/). :bow:

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-6183e5c9e2/how-to/). :bow:

**Comment by RackerKate** [comment]
@cyrichardson Please review.

**Comment by cyrichardson** [comment]
On it, @RackerKate.

**Comment by cyrichardson** [comment]
@RackerKate Suggestions here: 
[troubleshoot-a-cloud-block-storage-volume-in-read-only-mode_cyrEdits.docx](https://github.com/rackerlabs/rackspace-how-to/files/2885310/troubleshoot-a-cloud-block-storage-volume-in-read-only-mode_cyrEdits.docx)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-c3d9a42eff/how-to/). :bow:

File: content/cloud-block-storage/troubleshoot-a-cloud-block-storage-volume-in-read-only-mode.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+
+    # unmount /dev/xvdb1/
+
+The only output from this command is a command prompt.
+
+# Unmount a volume from a Windows server
+
+Use the following steps to unmount a volume from a Windows server:
+
+1. In Windows **Server Manager**, Select **File and Storage Services > Disks**.
```
Comment (catlook) [inline content/cloud-block-storage/troubleshoot-a-cloud-block-storage-volume-in-read-only-mode.md]: ... Manager**, select **File  ...

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-b55d78c9fa/how-to/). :bow:

File: content/cloud-block-storage/troubleshoot-a-cloud-block-storage-volume-in-read-only-mode.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+
+#### Make the volume persistent when the server restarts (Recommended)
+
+We recommend that you add the volume to the static file system information in
+your `fstab` configuration file. This step is optional, but it ensures that
+your volume remains attached to your server after the server restarts.
+
+Use the following steps to make the volume persistent:
+
+1. Open the **/etc/fstab** configuration file if it is not already open. The
```
Comment (catlook) [inline content/cloud-block-storage/troubleshoot-a-cloud-block-storage-volume-in-read-only-mode.md]: suggest: If it is not already open, open the **/etc/fstab** configuration file.

File: content/cloud-block-storage/troubleshoot-a-cloud-block-storage-volume-in-read-only-mode.md [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+   following example command opens it in the `nano` text editor:
+
+       root@nosnetdfw:~# nano /etc/fstab
+
+   The file should look like the following example:
```
Comment (catlook) [inline content/cloud-block-storage/troubleshoot-a-cloud-block-storage-volume-in-read-only-mode.md]: ... look similar to the following ...

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-400f656322/how-to/)

**Pull request merged**